qvm-get-image

```diff
@@ -21,15 +21,25 @@
 import argparse
 import qubesadmin
 import qubesimgconverter
+import sys
+
+# Maximum --allow-highres supported uncompressed image size =~ 15.6GB
```

Comment:
The size here and in the help text below do not match.

Reply:
The reason the size do not match with the size specified in help is discussed in detail with adw in previous messages. There is an independent barrier. i.e. `qubes.GetImageRGBA` uses `/tmp` to store temporary GraphicsMagick file. Which is  usually limited to 1GB in default templates. I have to make another commit to make the temporarily directory for `qubes.GetImageRGBA` configurable. But that would be to `qubes-core-agent-linux` repo.

I guess I have to write a better help to avoid confusion.